### ISSUE TITLE
Custom response

### ISSUE THREAD

**Issue by sumeko** [bshaffer/oauth2-server-php#607]
Is there a plan to make the response to be customize-able? 

and also would be great if the response speaks on our language.

**Comment by hellracer** [comment]
Yeah, I can concur that the response Content-Type is in  text/html;charset=UTF-8 is there a way to return in it application/json?

**Comment by hellracer** [comment]
I solved this issue by modifying the content-type on the reverse proxy side
